src/libraries/System.Private.Xml/src/System/Xml/Schema/XdrBuilder.cs

```diff
@@ -561,7 +561,7 @@ internal override bool IsContentParsed()
             return true;
         }
 
-        internal override void ProcessMarkup(XmlNode[] markup)
+        internal override void ProcessMarkup(XmlNode?[] markup)
```

Comment:
Why can't markup be null?

Reply:
Same answer as https://github.com/dotnet/runtime/pull/41488#discussion_r478771274.